.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -8,31 +8,7 @@ body:
     id: Description
     attributes:
       label: "Description"
-      description: Provide a brief explanation of the feature
-      placeholder: Describe in a few lines your feature request
+      description: Why do you want this sweet new feature and what does it address?
```

Comment:
This should probably be a different section. We're asking users why they want a feature before asking what the feature is. I do think it's a good question though, and probably deserves to be in the template.

Reply:
Yeah I think we just need one text area for them to fill out right? so like,
attributes:
      label: "Feature"
      description: Why do you want this sweet new feature and what does it address?
      placeholder: Tell us all about it!

I was trying to just simplify it to one area they can just slap text into. Were you thinking something else?

Reply:
It can totally be one required field like "What feature do you want added?", then an optional one like "Why do you want this feature?". It just helps for organization, as asking people to organize it themselves will probably lead to many different organizational styles.

Reply:
body:
  - type: textarea
    id: feature
    attributes:
      label: "Feature"
      description: What feature do you want added?
      placeholder: Tell us all about it!
    validations:
      required: true
  - type: textarea
    id: why_feature
    attributes:
      label: "Why do you want this feature?"
      placeholder: Cause it's awesome!
    validations:
      required: false

Reply:
Looks good!